docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: '3.8'
+
+services:
+  bot:
+    build: .
+    env_file: telegram-bot/.env
+    depends_on:
+      - mongo
+    volumes:
+      - .:/app
+    command: python telegram-bot/main.py
+
+  mongo:
+    image: mongo:latest
+    environment:
+      MONGO_INITDB_ROOT_USERNAME: root
+      MONGO_INITDB_ROOT_PASSWORD: example
+    ports:
+      - "27017:27017"
```

Comment:
Would like to get some comments about this from @CatOrLeader 

Maybe we can return to it and discuss during CI/CD and Deployment meeting

Reply:
Check the latest version of the compose file I pushed now